packages/api/endpoints/elasticsearch.js

```diff
@@ -177,10 +181,73 @@ async function changeIndex(req, res) {
   return res.send({ message });
 }
 
+async function indicesStatus(req, res) {
+  const esClient = await Search.es();
+
+  return res.send(await esClient.cat.indices({}));
+}
+
+async function indexFromDatabase(req, res) {
+  const esClient = await Search.es();
+
+  const indexName = req.body.indexName || timestampedIndexName();
```

Comment:
What if there is no req.body.indexName and timestampedIndexName() is incorrect? Like if it was a different date?

Reply:
I'm confused... can you give me an example?

Reply:
Or okay, I guess this is creating the index so using the current date is fine but what about referencing the index later on? How would they get the correct indexName?

Reply:
there is the indices-status call where you can view all your indices. You used to be able to see it the in AWS console, but when we put ES in the VPC you can't see all your indices in the console which is why I created that API call that just pretty much calls ES. Then the `current-index` API call shows you the one you're currently using. I think you also used to be able to see that through the console.

Reply:
👍 okay that's good